docs/Configuration.md

```diff
@@ -44,23 +44,12 @@ Below is an example for spark-default.conf, if you are using conda to install OA
 ##### Columnar Process Configuration
 
 spark.sql.sources.useV1SourceList avro
-spark.sql.join.preferSortMergeJoin false
-spark.plugins com.intel.oap.GlutenPlugin
+spark.plugins io.glutenproject.GlutenPlugin
 spark.shuffle.manager org.apache.spark.shuffle.sort.ColumnarShuffleManager
-
-# note Gluten Plugin depends on arrow data source
-spark.driver.extraClassPath $HOME/miniconda2/envs/oapenv/oap_jars/spark-columnar-core-<version>-jar-with-dependencies.jar:$HOME/miniconda2/envs/oapenv/oap_jars/spark-arrow-datasource-standard-<version>-jar-with-dependencies.jar
-spark.executor.extraClassPath $HOME/miniconda2/envs/oapenv/oap_jars/spark-columnar-core-<version>-jar-with-dependencies.jar:$HOME/miniconda2/envs/oapenv/oap_jars/spark-arrow-datasource-standard-<version>-jar-with-dependencies.jar
-
-spark.executorEnv.LIBARROW_DIR      $HOME/miniconda2/envs/oapenv
-spark.executorEnv.CC                $HOME/miniconda2/envs/oapenv/bin/gcc
+# optional clickhouse, gazelle-cpp
+spark.gluten.sql.columnar.backend.lib=velox
```

Comment:
The "=" should be removed.

Reply:
Is it better if we put the comment and the config item in the same line?
e.g. `spark.gluen.sql.columnar.backend.lib velox # Valid options: velox, clickhouse, gazelle-cpp`